packages/mcp-provider-code-analyzer/src/actions/run-analyzer.ts

```diff
@@ -25,10 +32,20 @@ export type RunInput = {
     target: string[]
 }
 
+type RunSummary = {
+    totalViolations: number
+    sev1Violations: number
+    sev2Violations: number
+    sev3Violations: number
+    sev4Violations: number
+    sev5Violations: number
+}
+
 // NOTE: THIS MUST ALIGN WITH THE ZOD SCHEMA DEFINED IN `sf-code-analyzer-run.ts`.
 export type RunOutput = {
     status: string
     resultsFile?: string
+    summary?: RunSummary
```

Comment:
Does this need to be optional? If so, do you have a test case covering the missing return value?